contents/docs/data/identify.mdx

```diff
@@ -105,8 +105,7 @@ curl -v -L --header "Content-Type: application/json" -d '{
 There are two requirements when assigning an `alias_id`:
 
 1. It cannot be associated with more than one `distinct_id`.
-2. The `alias_id` **must not** be the `distinct_id` of any other identified user. For example, assume we previously identified a user with `distinct_id_one` in a different session and device e.g., in [posthog-js](https://posthog.com/docs/libraries/js) using `posthog.identify('distinct_id_one')`. It is not possible to use `distinct_id_one` as an alias for any another distinct ID:
-
+2. The `alias_id` **must not** have been previously used as the `distinct_id` argument of an `identify()` or `alias()` call. For example: Assume we previously called `posthog.identify('distinct_id_one')`. It is not possible to use `distinct_id_one` as an alias ID:
```

Comment:
nit: maybe mention that one can check the UI to see if merge restrictions are applied or not.
note: this is stricter than reality and that's fine.

Reply:
Oh that's a great idea! It will be helpful for people who navigated to the doc from the tooltip in the app.